src/test/kotlin/org/rust/ide/docs/RsQuickDocumentationTest.kt

```diff
@@ -23,6 +23,7 @@ class RsQuickDocumentationTest : RsDocumentationProviderTest() {
     fun testGenericStructVariable() = checkDoc()
     fun testTupleDestructuring() = checkDoc()
     fun testConditionalBinding() = checkDoc()
+    fun testStructField() = checkDoc()
```

Comment:
Perhaps we should change this tests to use inline string literal fixtures instead of file fixtures? Would this be more convenient? One can mark string literals with `@Language("Rust")` or `@Langauge("html")` to get highlighting inside of the literals.